apps/frontend/components/___tests___/CheckField.spec.js

```diff
@@ -0,0 +1,38 @@
+import CheckField from "../CheckField";
+import React from "react";
+import { cleanup, fireEvent, render } from "react-testing-library";
+
+describe("<CheckField />", () => {
+  const baseProps = {
+    id: "root_foo",
+    label: "Foo Label",
+  };
+
+  afterEach(cleanup);
+
+  const props = {
+    ...baseProps,
+    schema: { format: "telephone", type: "boolean" },
+  };
+
+  it("click checkbox triggers input onChange callback", () => {
```

Comment:
@duranmla what about changing this to *triggers input onChange callback on click*